sssom/util.py

```diff
@@ -812,14 +851,25 @@ def curie_from_uri(uri: str, prefix_map: Mapping[str, str]) -> str:
 
 
 def get_prefixes_used_in_table(df: pd.DataFrame) -> List[str]:
+    """Get prefixes used in table.
```

Comment:
should write dataframe and not table. which parts of the dataframe does it get prefixes from?